.devcontainer/Dockerfile

```diff
@@ -5,6 +5,17 @@ FROM ubuntu:20.04
 # Instructs apt-get to run without a terminal
 ENV DEBIAN_FRONTEND=noninteractive
 
+# Terraform, providers and tflint versions
+ARG TERRAFORM_VERSION=1.0.0
+ARG AZURERM_VERSION=2.63.0
```

Comment:
If we're going to use azurerm 2.63, we'll need to update the version constraint in each `required_providers` block where 2.55 is specified (e.g. `/src/core/*/main.tf` and any of the `src/modules/*/main.tf` that reference it) and the UI Dockerfile

```
required_providers {
  azurerm = {
    source  = "hashicorp/azurerm"
    version = "= 2.63.0"
  }
  ...
}
```